ci/scripts/shipit

```diff
@@ -153,8 +158,8 @@ if [[ -z $(git config --global user.name) ]]; then
 fi
 
 pushd "${REPO_ROOT}"
-  for MANIFEST_PATH in $(ls manifests/*.yml); do
-    $DIR/update-manifest ${GITHUB_OWNER} ${RELEASE_NAME} ${VERSION} ${SHA1} ${MANIFEST_PATH}
+  for MANIFEST_PATH in manifests/*.yml; do
```

Comment:
cool, didn't know that worked!

Reply:
`shellcheck` told me I should not parse outputs of `ls` and instead use a glob :D

I didn't know 100% either.

Let's see if it works. I will put this into a modified pipeline that pulls this branch for the shipit task. I'd like to avoid 1h+ round trips + approvals to fix typos (like I made them yesterday)

Reply:
worked on my machine, should be fine! :) 👌